packages/web-components/src/components/va-combo-box/va-combo-box.scss

```diff
@@ -0,0 +1,59 @@
+@forward 'settings';
+
+@use 'usa-label/src/styles/usa-label';
+@use 'usa-error-message/src/styles/usa-error-message';
+@use 'usa-combo-box/src/styles';
+@use 'uswds-helpers/src/styles/usa-sr-only';
+@use 'usa-icon/src/styles/usa-icon';
+@use 'usa-select/src/styles/usa-select';
+@use 'usa-input/src/styles/usa-input';
+@use 'usa-hint/src/styles/usa-hint';
+
+@import '../../mixins/uswds-input-width.scss';
+@import '../../mixins/uswds-error-border.scss';
+
+
```

Comment:
We diverge a little bit with USWDS and use a yellow border on focus instead of blue so we'll want to do that with the combobox as well.

Here is an example of the yellow focus border on va-input:

![Screenshot 2024-11-01 at 9 17 01 AM](https://github.com/user-attachments/assets/74b2b6f1-3992-4268-9893-154cd39d35a1)

We have a mixin that you can import here should make that update for you:

```css
@import '../../mixins/focusable.css';
```

Reply:
awesome, will apply that here